src/cueUtilities.js

```diff
@@ -319,60 +320,83 @@ module.exports = function (params, cy, api) {
           }
         });
 
-        cy.on('tap', 'node', data.eTap = function (event) {
-          var node = this;
-          var opts = options();
-          let nodeHasCue = isSelectedGroupsContains(node);
-          if (hoveredGroup) {
-            nodeHasCue = nodeHasCue || hoveredGroup.id() === node.id();
+        function eventOnCueRegion(node, event) {
+          const expandcollapseRenderedStartX = node._private.data.expandcollapseRenderedStartX;
+          const expandcollapseRenderedStartY = node._private.data.expandcollapseRenderedStartY;
+          const expandcollapseRenderedRectSize = node._private.data.expandcollapseRenderedCueSize;
+          const expandcollapseRenderedEndX = expandcollapseRenderedStartX + expandcollapseRenderedRectSize;
+          const expandcollapseRenderedEndY = expandcollapseRenderedStartY + expandcollapseRenderedRectSize;
+
+          return event.offsetX >= expandcollapseRenderedStartX && event.offsetX <= expandcollapseRenderedEndX &&
+                  event.offsetY >= expandcollapseRenderedStartY && event.offsetY <= expandcollapseRenderedEndY;
+        }
+
+        function getGroupOfClickedOnCue(event) {
+          const clickedGroup = selectedGroups.find(group => eventOnCueRegion(group, event));
```

Comment:
Was this why you were asking about `null` vs `undefined` earlier? I think you can cleanup the return along the lines of `return clickedGroup ? clickedGroup : null;`

Reply:
Yup, this was why.